src/Analysis/Ast/Test/WorkspaceIndexManagerTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using FluentAssertions;
+using Microsoft.Python.Analysis.Indexing;
+using Microsoft.Python.Core.IO;
+using Microsoft.Python.Parsing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NSubstitute;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using TestUtilities;
+
+namespace Microsoft.Python.Analysis.Tests {
+    [TestClass]
+    public class WorkspaceIndexManagerTests : AnalysisTestBase {
+        private IDirectoryFileReader _rootFileReader;
```

Comment:
While current implementation of MSTest creates new instance of test class for every test, there is no guarantee that it won't change in the future. So I'd prefer to not use fields.

Reply:
I understand, but the fields get initialized on TestInitialize, so it doesn't depend on creating a new instance each time, I believe